src/txmempool.cpp

```diff
@@ -499,16 +499,17 @@ void CTxMemPool::removeRecursive(const CTransaction &origTx, MemPoolRemovalReaso
         RemoveStaged(setAllRemoves, false, reason);
 }
 
-void CTxMemPool::removeForReorg(const CCoinsViewCache *pcoins, unsigned int nMemPoolHeight, int flags)
+void CTxMemPool::removeForReorg(CChainState& active_chainstate, const CCoinsViewCache *pcoins, unsigned int nMemPoolHeight, int flags)
```

Comment:
There's no need to pass in a `CCoinsViewCache *` or `nMemPoolHeight` here any more. You can just get them from the `CChainState&`.

Reply:
Done in: 6dd22bc4b052f42cb9d102e940a4196d62c975fb
Much cleaner!